index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- ABOT HEADER -->
+    <header class="nav-bar">
+        <nav class="nav-section">
+            <p class="logo">My Logo</p>
+            <div class="btn">
+                <div class="btn-1"></div>
+                <div class="btn-1"></div>
+                <div class="btn-1"></div>
+            </div>
+        </nav>
+    </header>
+    <!-- ABOUT MYSELF -->
+    <main class="about">
+        <h2 class="about-title">I’m Martin.Glad to see you!</h2>
+        <p class="par-me">
+            I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
```

Comment:
- [ ] Kindly reduce the width and font-weight of this pargraph in accordance to the given design, reproducing perfectly a given design shows how well we pay attention to details. 👍 

## Your Design
<img width="256" alt="Untitled" src="https://github.com/mar4444/re-portfolio/assets/103386183/2ffac7a6-ba18-4a81-a5c8-0e87eb3e12ac">

## Figma Design
<img width="249" alt="Untitled" src="https://github.com/mar4444/re-portfolio/assets/103386183/e21b1f26-e640-4e66-b7b1-54a94126a9a0">